helix-term/src/main.rs

```diff
@@ -63,7 +59,7 @@ FLAGS:
         env!("CARGO_PKG_VERSION"),
         env!("CARGO_PKG_AUTHORS"),
         env!("CARGO_PKG_DESCRIPTION"),
-        logpath.display(),
+        log_path_default.display(),
```

Comment:
Shouldn't this show the overridden file if it is overridden?

Reply:
It can, will do!